src/static/js/modules/gallery.js

```diff
@@ -0,0 +1,42 @@
+function imageGallery() {
+    const ACTIVE_CLASS = "active";
+    const carouselSlide = document.querySelectorAll(".item-slide");
+    const previews = document.querySelectorAll(".gallery-img");
+    const reset = document.querySelector(".gallery-wrapper-miniture img")
+
+    previews.forEach((preview) => {
+        preview.addEventListener("click", function () {
+            let size = carouselSlide[0].clientWidth;
+            let counter = this.id;
+            previews.forEach((preview) => preview.classList.remove(ACTIVE_CLASS));
```

Comment:
use a different variable name so it is not mixed with the one you received from line 7.

Reply:
Done